apollo-router/src/router_factory.rs

```diff
@@ -581,16 +607,21 @@ pub(crate) async fn create_plugins(
                     .remove(name)
                     .unwrap_or_else(|| panic!("Apollo plugin not registered: {name}"));
                 if let Some(mut plugin_config) = $opt_plugin_config {
+                    // If any of the mandatory plugins need special treatment, then we'll
+                    // perform it here.
                     if name == "apollo.telemetry" {
                         // The apollo.telemetry" plugin isn't happy with empty config, so we
-                        // give it some. If any of the other mandatory plugins need special
-                        // treatment, then we'll have to perform it here.
+                        // give it some.
                         // This is *required* by the telemetry module or it will fail...
                         inject_schema_id(
                             Some(&Schema::schema_id(&schema.raw_sdl)),
                             &mut plugin_config,
                         );
                     }
+                    if name == "apollo.router_limits" {
```

Comment:
What special treatment would we need here? 👀

Reply:
ahh! shoot, I forgot to remove this; no special treatment, this was me going down a path that I've since abandoned